app/src/main/java/com/github/sdpsharelook/DBWord.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.sdpsharelook
+
+data class DBWord(
```

Comment:
This class is very similar to Word. It looks like a code duplication. I think there are much better solutions as adding constructors and fields to word

Reply:
I agree: it would be better if this was expressed in the serialization of the Word class.
However, not my job